dat/spob/delgon.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<spob name="Delgon">
+ <lua>default.lua</lua>
+ <pos x="-5718.691989" y="4217.135992"/>
+ <GFX>
+  <space>M23.webp</space>
+  <exterior>station03.webp</exterior>
+ </GFX>
+ <presence>
+  <faction>Proteron</faction>
+  <base>60.000000</base>
+  <bonus>0.000000</bonus>
+  <range>0</range>
+ </presence>
+ <general>
+  <class>M</class>
+  <population>10000</population>
+  <hide>1.000000</hide>
+  <services>
+   <land/>
+   <refuel/>
+   <bar/>
+   <missions/>
+  </services>
+  <commodities/>
+  <description>Another of the Proteron mining worlds, this world is largely mined by nanobots that the Proteron to send from space, and that suits them down to the ground.</description>
```

Comment:
```suggestion
  <description>Another of the Proteron mining worlds, this world is largely mined by nanobots that the Proteron send from space, and that suits them down to the ground.</description>
```
"that suits them down to the ground"?

Reply:
https://dictionary.cambridge.org/dictionary/english/suit-right-down-to-the-ground ? Not sure what you're saying here

Reply:
Wasn't aware that was a thing, but makes sense.
Do remove the to though.